src/evaluator/index.ts

```diff
@@ -4,9 +4,11 @@ import * as LabelsConstants from '../utils/labels';
 import { CONTROL } from '../utils/constants';
 import { ISplit, MaybeThenable } from '../dtos/types';
 import { IStorageAsync, IStorageSync } from '../storages/types';
-import { IEvaluationResult } from './types';
+import { IByFlagSetsResult, IEvaluationResult } from './types';
```

Comment:
You don't need to define a new interface `IByFlagSetsResult`. You can use `IEvaluationResult` because you don't have to calculate `elapsedMilliseconds` since telemetry latencies are handled in a different module.